docs/build-tools/sbt.md

```diff
@@ -49,11 +49,11 @@ addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "@BLOOP_VERSION@")
 Next, run:
 
 ```
-sbt "set bloopExportJarClassifiers := Some(Set(\"source\"))" bloopInstall
+sbt -Dbloop.export-jar-classifiers="sources" bloopInstall
```

Comment:
"source" was singular here, I think it was a typo.

Reply:
that was a typo that I previously lost 2 hours because of :sweat_smile: